.gitignore

```diff
@@ -397,3 +397,4 @@ FodyWeavers.xsd
 # JetBrains Rider
 *.sln.iml
 /StudyHub/StudyHub/appsettings.Development.json
+/StudyHub/StudyHub/appsettings.develop.json
```

Comment:
need to be removed and rename your local appsettings to **appsettings.Development.json**